### PR TITLE
Steve/access key UI improvement

### PR DESCRIPTION
**Pull request by SteveKok**

### CODE REVIEW

File: packages/frontend/src/components/profile/authorized_apps/AuthorizedApp.js [hunk 1]
```diff
@@ -59,6 +59,8 @@ const Container = styled.div`
 `;
 
 const AuthorizedApp = ({ app, onClick, deAuthorizing }) => {
+    console.log('app', app);
```
Comment (edwardchew97) [inline packages/frontend/src/components/profile/authorized_apps/AuthorizedApp.js:62]: Is the console.log intended here or shall we remove it?

**Review by edwardchew97 — APPROVED** [review]

File: packages/frontend/src/components/profile/authorized_apps/AuthorizedApp.js (hunk 1 above)
Comment (SteveKok) [reply packages/frontend/src/components/profile/authorized_apps/AuthorizedApp.js:62]: it is unintended, thanks for reviewing

File: packages/frontend/src/components/profile/full_access_key_rotations/FullAccessKeyRotation.js [hunk 2]
```diff
@@ -572,20 +572,29 @@ const FullAccessKeyRotation = ({ fullAccessKey }) => {
...
+                            {transactionHash.length > 15
+                                ? transactionHash
+                                      .substring(0, 6)
+                                      .concat(
+                                          '...',
+                                          transactionHash.substring(
+                                              transactionHash.length - 6
+                                          )
+                                      )
+                                : transactionHash}
```
Comment (Patrick1904) [inline packages/frontend/src/components/profile/full_access_key_rotations/FullAccessKeyRotation.js:595]: @SteveKok this code broke the /profile page while logged in. Please take a look ASAP.

Comment (SteveKok) [reply packages/frontend/src/components/profile/full_access_key_rotations/FullAccessKeyRotation.js:595]: @Patrick1904 thansk for informing us.

This problem is caused by sometimes indexer didn't give us the keyMeta and transaction hash when a new key is created.

We have solve this problem in this PR:
https://github.com/mynearwallet/my-near-wallet/pull/201/files

**Pull request merged**